### PR TITLE
Sow/resp hero section

### PR DESCRIPTION
**Pull request by abdallahsow**

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": []
```
Comment (ahamad-ch) [inline .vscode/extensions.json]: - [ ] What is this vscode related settings ?

File: styles.css [hunk 2]
```diff
@@ -511,4 +546,594 @@
...
+  }
+
+  .last-section {
+    justify-content: space-between;
+    gap: 0;
+    padding: 60px 50px;
+  }
+}
+
+@media screen and (max-width: 970px) {
```
Comment (ahamad-ch) [inline styles.css]: - [ ] In this PR as well, make sure to use standard breakpoints
```
576px
768px
992px
1024px
1200px
1440px
```

**Review by ahamad-ch — CHANGES_REQUESTED** [review]

File: .vscode/extensions.json (hunk 1 above)
Comment (abdallahsow) [reply .vscode/extensions.json]: Maybe while looking to fix prettier it poped up. I removed it

**Review by ahamad-ch — APPROVED** [review]
